v/vault/Dockerfiles/1.7.0-ubi8/Dockerfile.txt

```diff
@@ -2,7 +2,7 @@ FROM registry.access.redhat.com/ubi8
 
 MAINTAINER Mahima Gaikwad <mahima.gaikwad@ibm.com>
 
-ENV VAULT_VERSION=v1.7.0
+ARG VAULT_VERSION
```

Comment:
Can you provide default value for VAULT_VERSION as v1.7.0 ?
So that, if no build argument provided, this will take the default value.